src/RestBundle/Controller/RestController.php

```diff
@@ -42,9 +43,23 @@ public function indexAction(Request $request, $id)
         $this->_entityValidator = $this->get('validator');
         $this->_request = $request;
 
+        /* Current Route */
+        $route = $this->_request->get('_route');
+
+        /* Current Method */
+        $method = $this->_request->getMethod();
+
+        /* Current User */
+        $userToken = $this->_request->headers->get('x-user-token');
+        $userToken = 'notTheRealOne';
```

Comment:
This need to be removed in the future, for testing purposes, I am using this to avoid sending the header every time.